Gemfile

```diff
@@ -1,46 +0,0 @@
-source 'https://rubygems.org'
```

Comment:
Everything needed is baked into ChefDK - if the need arises I will create a new Gemfile (e.g. for deploying to supermarket with the stove gem), but until now it's just fluff.